From bb8bc44d22060cd1215712117cf30eae09f4f6ba Mon Sep 17 00:00:00 2001
From: Jappie Klooster <jappieklooster@hotmail.com>
Date: Fri, 2 Apr 2021 14:04:14 -0400
Subject: [PATCH] Fixup cmaklists

We remove cpm, which is a package manager for c++,
which requires networking, so it's better just deleted.

Furthermore we delete the adddirectory statements.
These want to modify directories outside of the sandbox.
---
 CMakeLists.txt | 26 --------------------------
 1 file changed, 26 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b5e8789..b797538 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,30 +13,6 @@ endif()
 
 include(${CPM_DOWNLOAD_LOCATION})
 
-CPMAddPackage(
-        NAME IODash
-        GITHUB_REPOSITORY YukiWorkshop/IODash
-        VERSION 0.1.0
-)
-
-CPMAddPackage(
-        NAME libevdevPlus
-        GITHUB_REPOSITORY YukiWorkshop/libevdevPlus
-        VERSION 0.2.1
-)
-
-CPMAddPackage(
-        NAME libuInputPlus
-        GITHUB_REPOSITORY YukiWorkshop/libuInputPlus
-        VERSION 0.2.1
-)
-
-CPMAddPackage(
-        NAME cxxopts
-        GITHUB_REPOSITORY jarro2783/cxxopts
-        VERSION 3.0.0
-        GIT_TAG 2d8e17c4f88efce80e274cb03eeb902e055a91d3
-)
 
 set(SOURCE_FILES_LIBRARY
         CommonIncludes.hpp
@@ -74,5 +50,3 @@ add_executable(ydotool ${SOURCE_FILES_CLIENT})
 target_link_libraries(ydotool ydotool_library dl pthread uInputPlus evdevPlus)
 install(TARGETS ydotool DESTINATION ${CMAKE_INSTALL_BINDIR})
 
-add_subdirectory(Daemon)
-add_subdirectory(manpage)
-- 
2.29.2

